### ISSUE TITLE
Release targets for v0.3.15 and python3.13 MacOS

### ISSUE THREAD

**Issue by Psy-Fer** [nanoporetech/pod5-file-format#149]
## Issue Description

There seems to be less targets in the latest v0.3.15 release. Also some python versions missing (3.13)

## Logs

On MacOS 13 and 14 using python3.13 which seems to be the system default?

ERROR: Could not find a version that satisfies the requirement lib-pod5==0.3.15 (from pod5) (from versions: none)
INFO: pip is looking at multiple versions of pod5 to determine which version is compatible with other requirements. This could take a while.
ERROR: No matching distribution found for lib-pod5==0.3.15

## Specifications

- Pod5 Version: 0.3.15
- Python Version: 3.13
- Platform: MacOS-13/MacOS-14

**Comment by 0x55555555** [comment]
Hi James,

Python 3.13 support is available in 0.3.21 - now live.

I hope that helps,
- George

**Comment by Psy-Fer** [comment]
Thanks George.

Is this

```
New end reason for reads terminated due to an analysis configuration change.
```

Something new or just the API request and device data error fields from a previous update?

Cheers 

James

**Comment by 0x55555555** [comment]
This is another new end reason added - but these aren't currently used in any live versions of MinKNOW.

**Comment by Psy-Fer** [comment]
Also will this version get pushed to PyPi?

Cheers,
James

**Comment by 0x55555555** [comment]
Hi James,

Yes - should be available now,

- George

**Comment by Psy-Fer** [comment]
Awesome.

Though it's failing to install h5py. It's using v3.11.0 which only works up to python3.12 

Same with pyarrow (though haven't looked at that yet.)

James

**Comment by Psy-Fer** [comment]
Looks like this is the culprit

https://github.com/nanoporetech/pod5-file-format/blob/master/python/pod5/pyproject.toml#L37C5-L37C42

There are no release targets for ~=3.11.0 for python3.13

**Comment by 0x55555555** [comment]
Hi @Psy-Fer ,

This should be resolved in 0.3.23 - the h5py requirement has been relaxed and should now work on python 3.13

- George

**Comment by Psy-Fer** [comment]
Hey George,

Yep that did the trick!

Thanks for getting that sorted out. All our tests are passing.

Cheers
James
